content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+These resources are designed to help users effectively respond to such scenarios, facilitating the development of thorough disaster recovery plans. By using these tools, teams can prepare to sustain stability and efficiency even in challenging situations. Check out our [documentation]() for more information.
+
+### IAM Policy Stream on Web Application
+
+In application development, accessing AWS resources like S3 buckets and RDS databases is common. To grant access, we create IAM roles and attach policies that specify permissions. However, determining the correct permissions can be challenging, often leading developers to assign excessive permissions to IAM roles. To address this, LocalStack introduced the **IAM Policy Stream**. This tool simplifies identifying the necessary permissions for cloud applications and helps detect logical errors.
+
+We have expanded the IAM Policy Stream feature to be accessible directly through the Web Application, in addition to the existing CLI feature. This enhancement will display the specific policy generated for each API call in the new interface, simplifying permission management and eliminating concerns about assigning correct permissions.
+
+The features include:
+
+1.  A real-time list of calls and the corresponding policies they generate.
+2.  A real-time summary policy that combines all individual policies into a single comprehensive policy.
+3.  The option to enable or disable this feature during runtime, allowing for performance optimization as needed.
+4.  The ability to reset the stream, enabling a fresh start with a new set of policies.
+
+{{< img-simple src="localstack-IAM-policy-stream.png" width=300 alt="Image of LocalStack IAM Policy Stream dashboard">}}
+
+Check out our [documentation]() and [video]() on getting started with the IAM Policy Stream.
+
+### Ephemeral Environments for LocalStack
+
+We have launched **Ephemeral Instances**, which allows you to run a LocalStack sandbox in the cloud, instead of your local machine. This ephemeral environment is a short-lived, encapsulated deployment of LocalStack which will be terminated after 90 minutes. With these sandboxes, you can run your tests, preview features in your AWS-powered applications, and collaborate asynchronously within and across your team!
+
+With Ephemeral Instances, you can use the same set of features that you use while running LocalStack locally, including the [Resource Browser](https://docs.localstack.cloud/user-guide/web-application/resource-browser/), [State Management (Cloud Pods)](), and [Extensions](https://docs.localstack.cloud/user-guide/extensions/)! While on your local machine, you can switch the [AWS Service Endpoint URL](https://docs.aws.amazon.com/general/latest/gr/rande.html) to point to the deployed sandbox URL to get started! Ephemeral Instances also allows you to generate an preview environment from GitHub Pull Request (PR) builds.
```

Comment:
might not capitalize 'Ephemeral Instance'

Reply:
links missing

Reply:
They will be available later